Sources/PersistenceService.swift

```diff
@@ -3,53 +3,55 @@ import Foundation
 struct PersistenceService: InterestPersistable {
 
     let service: UserDefaults
+    private let prefix = "com.pusher.sdk"
```

Comment:
Do we want to say `com.pusher.sdk.interests`, in case in the future we want the SDK to store other keys at the `com.pusher.sdk` level...

Reply:
Definitely not a bad idea 👍